src/org/jitsi/impl/neomedia/MediaStreamStatsImpl.java

```diff
@@ -1548,4 +1548,51 @@ public void addRembListener(REMBListener listener)
             }
         }
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * This method is different from {@link #getUploadRateKiloBitPerSec()} in
```

Comment:
I'll be commenting on the code inline. I consider these thinking-out-loud pieces of thoughts, not issues to be addressed.

I find the method name getUploadRateKiloBitPerSec... unappealing. Is it correct to have bit in there instead of bits. On top of that, its javadoc talks about download.